packages/mobile/package.json

```diff
@@ -5,17 +5,15 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "start": "react-native start",
-    "start:bg": "react-native start &",
+    "start": "./scripts/run_app.sh",
+    "start:android": "./scripts/run_app.sh -p android",
+    "start:ios": "./scripts/run_app.sh -p ios",
     "lint": "tslint -c tslint.json --project tsconfig.json",
     "build": "yarn run build:ts && yarn run build:metro",
     "build:sdk": "yarn --cwd ../walletkit build:for-env",
     "build:ts": "tsc --noEmit",
     "build:metro": "echo 'NOT WORKING RIGHT NOW'",
     "build:gen-graphql-types": "gql-gen --schema http://localhost:8080/graphql --template graphql-codegen-typescript-template --out ./typings/ 'src/**/*.tsx'",
-    "predev": "./scripts/pre-dev.sh",
-    "dev": "react-native run-android --appIdSuffix \"debug\" --no-packager && yarn start || echo 'Could not start metro server'",
-    "dev:ios": "react-native run-ios --simulator \"iPhone 11\"",
```

Comment:
I have a preference for keeping `dev:android` and `dev:ios` instead of `start:android` and `start:ios` as RN devs are too used to `start` being for starting only the packager (as in `react-native start`).

Reply:
Yep considered that too. I'm open to `dev`
I would have preferred `run` but that's a yarn keyword already